plugin-server/src/schema/hogflow.ts

```diff
@@ -91,9 +91,25 @@ const HogFlowActionSchema = z.discriminatedUnion('type', [
         type: z.literal('function_email'),
         config: z.object({
             message_category_id: z.string().uuid().optional(),
-            template_uuid: z.string().uuid().optional(), // May be used later to specify a specific template version
+            template_uuid: z.string().optional(), // May be used later to specify a specific template version
             template_id: z.literal('template-email'),
-            inputs: z.object({}),
+            inputs: z.object({
```

Comment:
Really want to get rid of this and swap to having each action validate itself using the inputs schema but we can come back to that